buildenv/jenkins/JenkinsfileBase

```diff
@@ -289,12 +289,13 @@ def post(test_target) {
 				}
 			}
 			//for performance test, achive regardless the build result
+			//TODO: Need to soft-code the output dir path
 			def benchmark_test_output_dir = 'jvmtest/performance/odm/ilog_wodm881/leftoverResults';
 			if (fileExists(benchmark_test_output_dir)) {
-				sh "tar -zcf ${testTarName} benchmark_test_output.tar.gz"
+				sh "zip -r benchmarkOutput.zip ${benchmark_test_output_dir}"
 				if (!params.ARTIFACTORY_SERVER) {
 					echo "ARTIFACTORY_SERVER is not set. Saving artifacts on jenkins."
-					archiveArtifacts artifacts: 'benchmark_test_output.tar.gz', fingerprint: true, allowEmptyArchive: true
+					archiveArtifacts artifacts: 'benchmarkOutput.zip', fingerprint: true, allowEmptyArchive: true
```

Comment:
Any reason for changing tar to zip. All the `*_test_output.tar.gz` will later upload to artifactory. This change breaks file upload later.

Reply:
I didn't notice this since `ARTIFACTORY_SERVER` wasn't set for my pipeline. Thanks for catching that. It's fixed now!